ntoskrnl/mm/section.c

```diff
@@ -4499,13 +4499,20 @@ MiRosUnmapViewInSystemSpace(IN PVOID MappedBase)
  *  instead of a memory backed section;
  *
  * File
- *  Unknown.
+ *  File object managed by CC manager.
  *
  * RETURN VALUE
  *  Status.
  *
  * @implemented
  */
+/*
+Use cases:
+1) !FileHandle && !FileObject: the section is backed by the paging file.
+2)  FileHandle && !FileObject: the section is backed by the specified file.
+3) !FileHandle &&  FileObject: the section is backed by the CC manager.
+4)  FileHandle &&  FileObject: (accepted, but should not happen!?).
+*/
```

Comment:
Since you're touching the documentation part of the API function, you might as well just write the documentation in Doxygen style.